packages/substrate-bindings/src/codecs/scale/ethAccount.ts

```diff
@@ -1,10 +1,10 @@
 import { fromHex, toHex } from "@polkadot-api/utils"
 import { Bytes, createCodec, createDecoder } from "scale-ts"
-import { keccak_256 as keccak } from "@noble/hashes/sha3"
+import { keccak_256 as keccak } from "@noble/hashes/sha3.js"
 
 const getFormattedAddress = (hexAddress: string) => {
   const nonChecksum = hexAddress.slice(2)
-  const hashedAddress = toHex(keccak(nonChecksum)).slice(2)
+  const hashedAddress = toHex(keccak(fromHex(hexAddress))).slice(2)
```

Comment:
I'm guessing this is fine, probably hexAddress is just `0x…`, but making sure this is on purpose: nonChecksum -> hexAddress?

`nonChecksum` sounds like it's excluding the checksum from hexAddress though, hence my confusion.

Reply:
Yeah, the naming is not very good.
`nonChecksum` should be a HexAddress that we don't know yet if it has (or does not have) a valid checksum. The change comes because @noble/hashes does not accept anymore HexString as input.